app/models/form_submission_attempt.rb

```diff
@@ -25,15 +25,31 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        user_uuid = form_submission.user_account.id
+        form_type = form_submission&.form_type
+        log_info = { form_submission_id:, benefits_intake_uuid: form_submission&.benefits_intake_uuid, form_type:,
+                     user_uuid: user_uuid }
+        zsf_function = "#{form_id} form submission to Lighthouse"
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        elsif should_send_form526_form4142_email
+          zsf_function = 'Form 526 Flow - Form 4142 failure email queuing'
+          form526_submission_id = Form526Submission.find_by(saved_claim_id:).id
+          Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify',
+                            log_info.merge({ form526_submission_id: }))
+          jid = EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail.perform_async form526_submission_id
```

Comment:
Wrap in `form526_submission_id` in parentheses for clarity and readability.

Reply:
Can you say some more about this? Not sure what is being asked here.

Reply:
For example: `Email.perform_async(id)`, instead of `Email.perform_async id`

Reply:
oh, yea this was showing up on a different line or something before. Yea I can do this, not sure why it isnt I usually always do.

Reply:
oh this is why 
```
app/models/form_submission_attempt.rb:41:121: C: Layout/LineLength: Line is too long. [121/120]
          jid = EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail.perform_async(form526_submission_id)
```

Reply:
addressed in 237835382b